### ISSUE TITLE
Hang on partprobe

### ISSUE THREAD

**Issue by roobyz** [clearlinux/clr-installer#277]
During installation I encountered a Clear Linux bug. Both fdisk and partprobe would freeze when run on my nvme drives.

**Steps to reproduce the behavior**:
1. Run clr-installer in TUI or batch mode
2. Install on /dev/sda
3. After installation process completes the formatting of partitions, the installation hangs
4. I noticed that "_partprobe /dev/nvme0n1_" was a hung process that prevents the installation from completing.

**Expected behavior**:
After completion of installation, system reboots into new clear linux

**Environment**:
 - Clear Linux OS Version: **28660**
 - Installer Version: **1.1.8**
 - Platform: Threadripper 1920x, X399 DESIGNARE EX, 128GB Ram, 128GB Intel SSD, 3x WD Black 1TB NVMe drives, AMD Vega20 (VII)

**Additional context**:
- Issue only experienced on clear linux (Ubuntu, Debian, and Manjaro all work fine)
- clr-installer does correctly configure the partitions in my install SSD 
- manually running partprobe without installation also hangs
- manually running fdisk to check the drive partitions also hangs
- running clr-installer with log level 4 shows that partprobe was the last process holding up the installation

**Comment by mdhorn** [comment]
@roobyz Can you share the model of the 1TB drive? And is it connected via SATA or M2?

**Comment by roobyz** [comment]
The WD Black 1TB NVMe drives are all PCIe M.2 2280 - WDS100T2X0C

**Comment by roobyz** [comment]
Tried installing again today with version 28750, which still hangs. I did notice that clear linux also hangs on _blkid_. Interestingly _partprobe_ (gnu parted tools), _fdisk_, and _blkid_ all cause the os to hang, but _lsblk_ does not, which apparently reads from sysfs directly. 

So it seems that the issue is somehow tied to when information is read from a nvme block device. Therefore the issue isn't tied directly to the installer, but to an issue within clear linux that is triggered by the installer or another process which tries to access the nvme block devices directly.
